packages/rainbowkit/src/wallets/walletConnectors/walletConnectWallet/walletConnectWallet.ts

```diff
@@ -21,9 +21,12 @@ export const walletConnectWallet = ({
     const ios = isIOS();
 
     const connector = getWalletConnectConnector({
-      projectId,
+      version: '1',
```

Comment:
I think we need to update the WalletConnectWalletOptions with the new `options` field, take that as a param in this `walletConnectWallet` fn, and then pass those additional options into this `getWalletConnectConnector` fn